src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,9 +5,35 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
+    private int cargoCapacity;
+    private int numberOfAxles;
+
+    public Truck(int cargoCapacity, int numberOfAxles) {
+        this.cargoCapacity = cargoCapacity;
+        this.numberOfAxles = numberOfAxles;
+    }
+
     public Truck() {
     }
 
+    public int getCargoCapacity() {
+        return cargoCapacity;
+    }
+
+    public Truck setCargoCapacity(int cargoCapacity) {
+        this.cargoCapacity = cargoCapacity;
+        return this;
+    }
+
+    public int getNumberOfAxles() {
+        return numberOfAxles;
+    }
+
+    public Truck setNumberOfAxles(int numberOfAxles) {
+        this.numberOfAxles = numberOfAxles;
+        return this;
+    }
+
     @Override
```

Comment:
Overriding a method should typically include a comment, even if it's a simple one, to explain the specific behavior of the method in the current class context.